image/Makefile

```diff
@@ -4,7 +4,9 @@ CLI_DIR:=$(CURDIR)/../../cli
 VERSION?=0.0.0-dev
 STATIC_VERSION=$(shell ../static/gen-static-ver $(ENGINE_DIR) $(VERSION))
 DOCKER_HUB_ORG?=docker
+ARCH=$(unamae -m)
```

Comment:
`s/unamae/uname/`

Reply:
So I can demonstrate that part of this works. 

I can call `make release` locally and that should build an x86 image. 
Then I can run `make manifest` but I would need to modify the `build-manifest-list` script since I only have an x86 image.